### ISSUE TITLE
Tab crashed while dropping images

### ISSUE THREAD

**Issue by pdewouters** [humanmade/h2#335]
## Issue Description

As I dropped a photo into the comment box on a weekly update post, the whole page went blank and Firefox dev edition crashed.

<img width="925" alt="screenshot 2019-02-04 at 09 51 47" src="https://user-images.githubusercontent.com/30460/52201360-1cb4e000-2863-11e9-9330-4173e940fc5e.png">
<img width="291" alt="screenshot 2019-02-04 at 09 51 17" src="https://user-images.githubusercontent.com/30460/52201361-1cb4e000-2863-11e9-912e-aca9eb9aef44.png">
<img width="470" alt="screenshot 2019-02-04 at 09 50 26" src="https://user-images.githubusercontent.com/30460/52201386-276f7500-2863-11e9-8dd0-e920f346c6a0.png">

## Steps to reproduce:

I tried to reproduce but couldn't.
Basically, I typed my update then dragged and dropped photos from the Photos app.

## Expected Result:

I expect the photos to be posted and not crash the browser tab.
